connect/runtime/src/main/java/org/apache/kafka/connect/runtime/AbstractHerder.java

```diff
@@ -106,6 +116,12 @@ public void onStartup(String connector) {
     }
 
     @Override
+    public void onPause(String connector) {
```

Comment:
Doesn't look like there's a great reason. It's been a while since I wrote this, but I guess I was thinking that `onResume` made less sense for a Connector since there's no difference between starting and resuming. But that would suggest we shouldn't have `onPause` as well, so it might have just been to simplify the implementation.

Another question I was thinking about is whether connectors are "restartable" as I'm assuming for this implementation. Is it ok to call start() after calling stop() without the initial call to initialize()?